cardinal/txqueue/queue.go

```diff
@@ -1,8 +1,10 @@
-package message
+package txqueue
```

Comment:
nit: after some thought, i think "txpool" is technically more correct here because tx is processed in batched not FIFO